src/lib/moduleResolver.ts

```diff
@@ -14,18 +15,30 @@ export interface ToAbsMid {
 	(moduleId: string): string;
 }
 
+export const RESOLVE_CONTENTS = Symbol();
+
 export default function makeResolver(toAbsMid: ToAbsMid): ResolveMid {
-	return function resolveMid<T>(mid: string): Promise<T> {
+	return function resolveMid<T>(mid: string, member: string | symbol = 'default'): Promise<T> {
 		return new Promise((resolve) => {
 			// Assumes require() is an AMD loader!
-			require([toAbsMid(mid)], (module) => {
-				if (module.__esModule) {
-					resolve(module.default);
-				}
-				else {
-					resolve(module);
+			require([toAbsMid(mid)], resolve);
```

Comment:
Looks like `dojo-core/load` falls back to the available `require` method, so that would work for modules right? That would be a follow-up change though.